lib/cookbook-omnifetch/artifactory.rb

```diff
@@ -93,13 +98,13 @@ def to_lock
         "#to_lock must be implemented on #{self.class.name}!"
     end
 
-    # The path where all pristine tarballs from an artifactserver are held.
+    # The path where all pristine tarballs from an artifactory are held.
     # Tarballs are moved/swapped into this location once they have been staged
     # in a co-located staging directory.
     #
     # @return [Pathname]
     def cache_root
-      Pathname.new(CookbookOmnifetch.cache_path).join(".cache", "artifactserver")
+      Pathname.new(CookbookOmnifetch.cache_path).join(".cache", "artifactory")
```

Comment:
It seems like this would invalidate the cache for folks who were using a previous version of this - is that a concern?

Reply:
This was copy/pasted from the artifactserver (SuperMarket) implementation, so it was writing the content in the wrong place anyway. Impact should be neglible since this was broken for Artifactory users anyway.